docs/monthly-meeting/2022-05.md

```diff
@@ -0,0 +1,145 @@
+
+Carol: we don't want to wholesale restructure the docs.  instead, supplement what we have and incrementally change what we have.
+
+This group shouldn't discuss working group mechanics.
+
+CAM: To reduce confusion: if the main point of the WG is to be the editorial board, maybe a new name would make it clearer.
+
+Guido: renaming could be a good idea. Let the WG form their own meeting schedule. Petr was saying he didn't want to be on the WG, he wanted to be in the Docs community. Petr proposed using diataxis, which doesn't recommend large-scale restructuring, so it's in line with Carol's recommendations also.
+
+Lightweight governance is best.
+
+- [ ] Carol (and CAM?) will be updating the readme and guide at the sprints.
+
+If we rename the WG to an Editorial Board, we should clarify the interpretation of the name.
+
+A quarterly meeting for the WG seems like the right cadence.
+
+Other possible names:
+- Docs Steering Council.
+- Docs Advisory Board.
+- Docs Oversight Board.
+
+The current members of the WG will discuss their name.  It's not a one-way decision, so we can work on it.
+
+Editorial Board is the agreed name. Wikipedia has a good definition which aligns with our meaning.
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, &c.*
+
+#### Style Guide
+
+Ned: Would like to have a clearer single voice, instead of pages having disparate content types in terms of reference, examples, narrative style etc. Consistency on "tone of voice" (neutral "the", plural "we"), structuring of invidiual pages, is somewhat lacking. Discussing or deciding on a single "house style" for documentation pages may be a useful thing to do to produce a reference.
+
+Guido: A north star document would be useful, but he sees a risk of "PEP 8" like changes -- mechanical edits to the text without substantive improvements
+
+Carol: There is a style guide in the DevGuide, although very dated. Mariatta thinks it should be held somewhere that isn't the DevGuide, (didn't catch where?).
+
+There's a current style guide in the devguide -- we should review this one and check what (if anything) we should improve and update.
+
+- [ ] Anyone can make pull requests to update the devguide.  Carol & Mariatta can review pull requests.
+
+#### Improving documentation PR review
+
+Ned: Concept of Documentation-focused committers? A pool of people who can merge Docs PRs. The Jupyter team set out guidance for when documentation should be changed.
+For now we should focus on substantive changes whilst we have fewer committers focussed on docs.
+
+Adam: can we have a team of people who are "approved" to review docs changes? Improving the reviewing story for docs, so that core devs can use it as a signal that someone trustworthy has given a +1 to the change.
+
+- [ ] Mariatta will open a ticket to give CAM, Pradyun, and Adam triage permissions for CPython.
```

Comment:
```suggestion
- [x] Mariatta will open a ticket to give Ned, CAM, Pradyun, and Adam triage permissions for CPython.
```

Reply:
Sorry to omit you @nedbat -- I had assumed you were already a triager!

A